Exercise_4/docs/part-3.md

```diff
@@ -0,0 +1,176 @@
+![uscore_uscdi_mapping](img/uscore_uscdi_mapping.png)
+
+The complete table is available here:
+[https://www.hl7.org/fhir/us/core/general-guidance.html](https://www.hl7.org/fhir/us/core/general-guidance.html)
+
+### Requirements and elements to be US Core compliant
+The complete list of **US Core Profiles** is below. Each profile defines the minimum mandatory elements, extensions and terminology requirements that **MUST** be present to satisfy that profile. 
+
+For each profile, requirements and guidance are given in a simple narrative summary.
+??? Note "Complete list of US Core Profiles"
+    * US Core AllergyIntolerance Profile
+    * US Core BMI Profile
+    * US Core Blood Pressure Profile
+    * US Core Body Height Profile
+    * US Core Body Temperature Profile
+    * US Core Body Weight Profile
+    * US Core CarePlan Profile
+    * US Core CareTeam Profile
+    * US Core Condition Profile
+    * US Core DiagnosticReport Profile for Laboratory Results Reporting
+    * US Core DiagnosticReport Profile for Report and Note exchange
+    * US Core DocumentReference Profile
+    * US Core Encounter Profile
+    * US Core Goal Profile
+    * US Core Head Circumference Profile
+    * US Core Heart Rate Profile
+    * US Core Immunization Profile
+    * US Core Implantable Device Profile
+    * US Core Laboratory Result Observation Profile
+    * US Core Location Profile
+    * US Core Medication Profile
+    * US Core MedicationRequest Profile
+    * US Core Organization Profile
+    * US Core Patient Profile
+    * US Core Pediatric BMI for Age Observation Profile
+    * US Core Pediatric Head Occipital-frontal Circumference Percentile Profile
+    * US Core Pediatric Weight for Height Observation Profile
+    * US Core Practitioner Profile
+    * US Core PractitionerRole Profile
+    * US Core Procedure Profile
+    * US Core Provenance Profile
+    * US Core Pulse Oximetry Profile
+    * US Core Respiratory Rate Profile
+    * **US Core Smoking Status Observation Profile**
+    * US Core Vital Signs Profile
+
+You may have noticed that **SMOKING STATUS** again! It was in the EHR data we saw, and it was in the USCDI criteria... we’ll revisit this example soon...
+
+### IG navigation example by detailed US Core walkthrough
+
+Let’s look specifically at the US Core Medication Profile here.
+![uscore_medications](img/uscore_medications.png)
+
+We can see details of the elements of the medication profile: [http://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-medication.html](http://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-medication.html)
+
+??? question "Knowledge Check: Which element(s) of this profile must be supported in order to be US Core Compliant?"
+    `Medication.code` element
```

Comment:
Here you make a knowledge check without explaining the Profile view, that each field has a:
- name
- flags (including "Is Modifier", "Is Summary", "Is Must Support")
- cardinality
- data type
- description and constraints

We need to explain what they are seeing before you ask them what they saw.